GOVERNANCE.md

```diff
@@ -0,0 +1,192 @@
+# Atlantis Governance
+
+**Atlantis** is committed to building an open, inclusive, productive, and self-governing community focused on building a high-quality infrastructure orchestration system. This document governs the community and defines how it should work together to achieve this goal.
+
+Atlantis follows a two-tier governance model. The higher tier comprises the Atlantis Steering Committee, responsible for the project's overall health. [Maintainers](#Maintainer), [Core-Contributors](#Core-Contributor), and [Members](#Member) make up the lower tier. They are the main contributors to one or more repositories within the overall project.
+
+The governance policies defined here apply to all repositories in the runatlantis GitHub organization.
+
+## Atlantis Steering Committee
+
+The Atlantis project has a project steering committee consisting of 5 members, with a maximum of 1 member from any single organization. The steering committee in Atlantis has a final say in any decision concerning the Atlantis project, with the exceptions of deciding steering committee membership, and changes to project governance. See [Updating Governance](#Updating-Governance).
```

Comment:
does it have to be 5 members off the start, or we can grow into it once we start our cncf journey?